mlrun/__init__.py

```diff
@@ -11,18 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""
-MLRun is a generic and convenient mechanism for data scientists and software developers to describe and run tasks
-related to machine learning (ML) in various, scalable runtime environments and ML pipelines while automatically
-tracking executed code, metadata, inputs, and outputs.
-MLRun integrates with the `Nuclio <https://nuclio.io/>`_ serverless project and with `Kubeflow
-Pipelines <https://github.com/kubeflow/pipelines>`_.
-
-The MLRun package (``mlrun``) includes a Python API library and the ``mlrun`` command-line interface (CLI).
-"""
```

Comment:
Are you sure placing a docstring in the package's `__init__` is not some kind of convention ?
e.g. (just throwing a guess) maybe `pip` / `pypi` are using it for something ?

Reply:
pypi takes what we point it to (readme.md), i wanted to separate between API ref doc and MLRun overview docs, if i keep it it will be present in the specific package doc header which is not the right place for it